core/src/test/java/hudson/FilePathTest.java

```diff
@@ -464,7 +464,7 @@ private void checkTarUntarRoundTrip(String filePrefix, long fileSize) throws Exc
     }
 
     @Test public void symlinkInTar() throws Exception {
-        if (Functions.isWindows())  return; // can't test on Windows
+        assumeTrue(!Functions.isWindows()); // can't test on Windows
```

Comment:
https://github.com/jenkinsci/jenkins/blob/master/pom.xml#L151-L155 in 4.11 exists
Right after `assumeTrue` and before `assumeThat`